drivers/display/display_sdl_bottom.c

```diff
@@ -8,15 +8,16 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <stdbool.h>
+#include <autoconf.h>
 #include <SDL.h>
 #include "nsi_tracing.h"
 
 int sdl_display_init_bottom(uint16_t height, uint16_t width,
 			    void **window, void **renderer, void **texture)
 {
 	*window = SDL_CreateWindow("Zephyr Display", SDL_WINDOWPOS_UNDEFINED,
-				   SDL_WINDOWPOS_UNDEFINED, width,
-				   height, SDL_WINDOW_SHOWN);
+				   SDL_WINDOWPOS_UNDEFINED, width * CONFIG_SDL_DISPLAY_ZOOM_PCT / 100,
+				   height * CONFIG_SDL_DISPLAY_ZOOM_PCT / 100, SDL_WINDOW_SHOWN);
```

Comment:
Not an SDL expert, but wouldn't it make more sense to use [`SDL_RenderSetScale`](https://wiki.libsdl.org/SDL2/SDL_RenderSetScale)?

Reply:
Me neither, but something as trivial as this doesn't work:

```c
        /* hard-coded factors, just for testing purpose */
        if (SDL_RenderSetScale(*renderer, 2.0, 2.0) != 0) {
                nsi_print_warning("Failed to set SDL render scale: %s", SDL_GetError());
                return -1;
        }
```